src/d2.js

```diff
@@ -8,6 +8,7 @@ import I18n from './i18n/I18n';
 import Config from './config';
 import CurrentUser from './current-user/CurrentUser';
 import { fieldsForSchemas } from './model/config';
+import { dataStore } from './datastore/DataStore';
```

Comment:
It doesn't really matter, but since `dataStore` is the default export in `datastore/DataStore.js`, the `{}` is not required here. You could instead do:

```
import DataStore from './datastore/DataStore';
```